packages/extension/app/background.js

```diff
@@ -19,6 +20,8 @@ const firstOrNull = (cookies) => {
 }
 
 const connect = function (host, path, extraOpts) {
+  const isChromeLike = !!window.chrome && !window.browser
```

Comment:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/9cd24901f983e703d9032a251ad3d60f8357970f. Had to flip the logic since Firefox is the only implementation of `getBrowserInfo`, so it's the only one where we know what name to check against.